packages/wethegit-components/src/components/in-view/components/in-view-item/styles/in-view-item.module.scss

```diff
@@ -55,10 +55,10 @@
 // Example: `.duration15` is a 1.5s duration.
 @for $i from 0 through 20 {
   .duration#{$i} {
-    --duration: #{get-duration($i, 20)};
+    --duration: #{get-duration($i)};
   }
 
   .staggerDuration#{$i} {
-    --stagger-duration: #{get-duration($i, 20)};
+    --stagger-duration: #{get-duration($i)};
   }
```

Comment:
These two lines were the culprit (passing `20` for the second arg). I ended up removing the second arg entirely though.